anyTV/eslint-config-backend
# Dev-5061 Moments > Code Maintenance > New code at 300 Max lines

[Dev-5061](https://freedom.myjetbrains.com/youtrack/issue/Dev-5061) Moments > Code Maintenance > New code at 300 Max lines


Test

1. go to moments backend folder
2. Create a new branch git checkout -b test-max-len
3. npm i github:anyTV/eslint-config-backend#feat-update-eslint-backend --save-dev
5. Try to edit a file by adding lots of console.log until it reaches more than 300 lines
6. Try to commit the file
7. Create a PR
8. on the PR trigger eslint by assigning a reviewer
9. New rule max lines 300 should appear and prevent block the merging
